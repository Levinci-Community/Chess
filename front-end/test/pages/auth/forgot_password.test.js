import React from "react";
import { render } from "@testing-library/react";
import ForgotPasswordPage from "../../../src/pages/auth/forgot_password";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({
    t: (key) => key,
  }),
}));

test("renders ForgotPasswordPage snapshot", () => {
    const { container } = render(
      <Router>
        <ForgotPasswordPage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
