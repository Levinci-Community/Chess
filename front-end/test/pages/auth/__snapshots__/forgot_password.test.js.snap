// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ForgotPasswordPage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-25ehpw"
  >
    <div
      class="chakra-stack css-1821gv5"
    >
      <div
        class="chakra-stack css-8g8ihq"
      >
        <div
          class="chakra-stack css-tl3ftk"
        >
          <div
            class="chakra-stack css-1igwmid"
          >
            <img
              alt="common.ute_chess_club"
              class="chakra-image css-1570kgy"
              src="/logo.png"
            />
            <p
              class="chakra-text css-1hf07bm"
            >
              common.ute_chess_club
            </p>
          </div>
        </div>
      </div>
      <div
        class="css-1b0byzm"
      >
        <div
          class="chakra-stack css-mbssy4"
        >
          <div
            class="chakra-stack css-5bh0zh"
          >
            <h2
              class="chakra-heading css-0"
            >
              auth.forgot_password
            </h2>
          </div>
          <div
            class="chakra-stack css-1gnc3qn"
          >
            <div
              class="chakra-form-control css-0"
              role="group"
            >
              <label
                class="chakra-form__label css-uvho5g"
                for="email"
                id="field-:r0:-label"
              >
                 
                auth.email
              </label>
              <input
                aria-required="true"
                class="chakra-input css-0"
                id="email"
                required=""
                type="email"
              />
            </div>
          </div>
          <div
            class="chakra-stack css-mbssy4"
          >
            <button
              class="chakra-button css-4xx2wk"
              type="button"
            >
              auth.reset_password
            </button>
            <hr
              aria-orientation="horizontal"
              class="chakra-divider css-1upb9tn"
            />
            <p
              class="chakra-text css-bmfe2s"
            >
              auth.remember_password
              <a
                class="chakra-link css-1o61trh"
                href="/login"
              >
                auth.login
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
